### PR TITLE
migrate NFT arbitrum models

### PR DESCRIPTION
**Pull request by 0xRobin**
Migrates Arbitrum NFT models

### CODE REVIEW

**Review by jeff-dude — COMMENTED** [review]
code looks 🔥 per usual, but curious why you made tofu_v1 a macro, if it's not exactly set up to be used for multiple models? the macro itself hardcodes version value and such. is this something you plan to enhance in the future, as more versions are released?

File: models/_sector/nft/trades/old/platforms/stealcam_arbitrum_events.sql [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {{ config(
     schema = 'stealcam_arbitrum',
     alias = alias('events'),
-    partition_by = ['block_date'],
```
Comment (jeff-dude) [inline models/_sector/nft/trades/old/platforms/stealcam_arbitrum_events.sql:4]: also, how come we're removing partitions on the arbritrum spells?

Comment (0xRobin) [reply models/_sector/nft/trades/old/platforms/stealcam_arbitrum_events.sql:4]: following this guideline on delta lake partitioning:
https://docs.databricks.com/tables/partitions.html#do-small-tables-need-to-be-partitioned
I'm going to remove partitioning from the smaller marketplace models. (this specific one is only 50k records)

**Comment by 0xRobin** [comment]
> if it's not exactly set up to be used for multiple models?

The macro will be re-used for tofu v1 models on bnb, polygon and optimism :)

File: models/_sector/nft/trades/old/platforms/stealcam_arbitrum_events.sql (hunk 1 above)
Comment (jeff-dude) [reply models/_sector/nft/trades/old/platforms/stealcam_arbitrum_events.sql:4]: good point, we should eventually do this check universally. i'll add to list for post-migration.

**Pull request merged**